src/app/docs/split-view/split-view-docs.component.html

```diff
@@ -294,7 +294,7 @@
         heading="Automatically advance to the next item"
       >
         <p class="sky-docs-text-block">
-          If users are likely to work through the list sequentially, automatically advance to the next item after users invoke the primary action in the workspace.
+          If users are likely to work through the list sequentially, remove the selected item and automatically advance to the next item after users invoke the primary action in the workspace.
```

Comment:
This looks good to me, but I'd like to get @Blackbaud-AlexKingman to check the update too since he's the one who looked into this and filed the issue.

The only suggestion I have is to possibly consider adding another sentence to the end of the existing sentence to explain to `Always display the active element at the top of the list.` That said, I'm not sure if we want that or not. Reading through Alex's comment at https://dev.azure.com/blackbaud/Products/_workitems/edit/1773203, it seems like maybe we do. But I'm not 100 percent sure and would also like his feedback on that.

Reply:
Thanks, John.

Reply:
I'd defer to UX for that answer. @Blackbaud-ToddRoberts any thoughts?

Reply:
i don't think we should add that as a guideline - if someone jumps ahead a few, we don't want to autoscroll that item to the top. it's sufficient to remove the item when they take action on it as the next item will always be in view that way. 

We do already have guidance on the modern split view page template covering removing the item: 
> When users take action on an item, remove that item from the list and open the next item in the workspace.

Reply:
Sounds good. Seems like we should keep the first change that Paula already made, which is consistent with that modern theme guideline, and not worry about the extra sentence. Thanks!